### PR TITLE
Move Samples to Resolwe-Bio

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -78,6 +78,7 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': pgname,
         'USER': pguser,
+        'PASSWORD': 'genesis',
```
Comment (tjanez) [inline tests/settings.py]: @dblenkus, did this unintentionally slip into the commit?

Comment (dblenkus) [inline tests/settings.py]: Tnx :)

**Comment by tjanez** [comment]
Please, add a CHANGELOG entry

**Comment by mstajdohar** [comment]
LGTM, @dblenkus please merge.

**Pull request merged**